### PR TITLE
Rename `eth1/` -> `el/` and `eth1_monitor.nim` -> `el_monitor.nim`

### PR DESCRIPTION
**Pull request by henridf**

### CODE REVIEW

**Comment by tersec** [comment]
Is the module bump an intrinsic part of this, or can be it separated?

**Comment by henridf** [comment]
> Is the module bump an intrinsic part of this, or can be it separated?

oops the module bump is not needed, these are some local changes that got swept up in my commit.

**Comment by tersec** [comment]
```
[2023-05-12T14:50:24.567Z] + make DISABLE_TEST_FIXTURES_SCRIPT=1 test
[2023-05-12T14:50:24.568Z] Building: build/consensus_spec_tests_minimal
[2023-05-12T14:50:24.568Z] Building: build/consensus_spec_tests_mainnet
[2023-05-12T14:56:07.367Z] Build completed successfully: build/consensus_spec_tests_minimal
[2023-05-12T14:56:07.367Z] Building: build/nimbus_signing_node
[2023-05-12T14:56:07.367Z] Build completed successfully: build/consensus_spec_tests_mainnet
[2023-05-12T14:56:07.367Z] Building: build/state_sim
[2023-05-12T14:58:02.592Z] Build completed successfully: build/state_sim
[2023-05-12T14:58:02.592Z] Building: build/block_sim
[2023-05-12T14:58:37.549Z] Build completed successfully: build/nimbus_signing_node
[2023-05-12T15:00:02.527Z] Build completed successfully: build/block_sim
[2023-05-12T15:00:02.527Z] Building: build/all_tests
[2023-05-12T15:00:44.188Z] /home/jenkins/workspace/2_platforms_linux_x86_64_PR-4944/tests/all_tests.nim(26, 3) Error: cannot open file: ./test_el_manager
[2023-05-12T15:00:44.188Z] make: *** [Makefile:345: all_tests] Error 1
script returned exit code 2
```
https://ci.status.im/blue/organizations/jenkins/nimbus-eth2%2Fplatforms%2Flinux%2Fx86_64/detail/PR-4944/4/pipeline

**Review by tersec — APPROVED** [review]

**Comment by henridf** [comment]
Jenkins failed with `[2023-05-12T19:16:04.404Z] Geth 0 failed to start`. Rebased on unstable, which will kick off a new run.

File: beacon_chain/rpc/rest_nimbus_api.nim [hunk 1]
```diff
@@ -223,12 +223,12 @@ proc installNimbusApiHandlers*(router: var RestRouter, node: BeaconNode) =
         updateLogLevel(level)
     return RestApiResponse.jsonResponse((result: true))
 
-  router.api(MethodGet, "/nimbus/v1/eth1/chain") do (
+  router.api(MethodGet, "/nimbus/v1/el/chain") do (
```
Comment (etan-status) [inline beacon_chain/rpc/rest_nimbus_api.nim]: mmh, would avoid randomly renaming API endpoints as part of a file-level rename

Comment (henridf) [reply beacon_chain/rpc/rest_nimbus_api.nim]: oops totally ! I applied `sed` too bluntly and didn't even notice I had changed that 🤦

**Comment by henridf** [comment]
Current failure in "Connections pool stress test" seems unrelated.

**Pull request merged**